prac_05/state_names.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+def print_names():
+    for code,name in CODE_TO_NAME.items():
+        print(f"{code:3}: {name}")
```

Comment:
Can the length of the code be adjusted dynamically? What if there was a brand new state with a code longer than 3 characters?